src/pages/Inventory.page.js

```diff
@@ -9,7 +9,49 @@ export class InventoryPage extends BaseSwagLabPage {
 
     get addItemToCartBtns() { return this.page.locator('[id^="add-to-cart"]'); }
 
+    inventoryItemsName = '.inventory_item_name';
+
+    inventoryItemsPrice = '.inventory_item_price';
+
+    get sort() { return this.page.locator('.product_sort_container'); }
+
+    get activeOption() { return this.page.locator('.active_option'); }
+
+    async newSortItem(value) {
+        await this.page.selectOption('.product_sort_container', value);
+    }
+
+    async getActiveOptionText() {
+        return this.activeOption.textContent();
+    }
+
     async addItemToCartById(id) {
         await this.addItemToCartBtns.nth(id).click();
     }
+
+    async getInventoryItemsPrices() {
+        const prices = await this.page.$$eval(this.inventoryItemsPrice, (elements) => elements.map((element) => element.textContent));
```

Comment:
you can use [allTextContent](https://playwright.dev/docs/api/class-locator#locator-all-text-contents) to get elements text. This means you don't need to use $$eval as playwright discourage using it: 
<img width="967" alt="image" src="https://github.com/BohdanObruch/playwright_tests_with_po/assets/30896478/92dec13a-0670-4629-8065-53bad94b0230">